runtime/parser/type.go

```diff
@@ -921,11 +921,30 @@ func defineIdentifierTypes() {
 					return nil, err
 				}
 
-				entitlements, err := parseEntitlementList(p)
-				if err != nil {
-					return nil, err
+				p.skipSpaceAndComments()
+
+				keyword := p.currentTokenSource()
+				switch string(keyword) {
+				case KeywordMapping:
+					keywordPos := p.current.StartPos
+					// Skip the keyword
+					p.nextSemanticToken()
+
+					entitlementMapName, err := parseNominalType(p, lowestBindingPower)
+					if err != nil {
+						return nil, err
+					}
+					authorization = ast.NewMappedAccess(entitlementMapName, keywordPos)
+					p.skipSpaceAndComments()
```

Comment:
Is this needed?

Reply:
Yes, nominal type parsing does not skip space/comments at the end